### ISSUE TITLE
[question] Release early, release often?

### ISSUE THREAD

**Issue by riga** [rootpy/rootpy#675]
It has been quite some time since the last version of rootpy was released.
Are there any efforts to push a new release, e.g. 0.8.1?

In our software setup, we prefer packages from pypi over source installations
from master branches, although it would not be a big issue in the case of rootpy.

But anyway, is there a reason for the low-frequency release cycle?

**Comment by ndawe** [comment]
Yeah, I can release 0.8.1 now. A few bugs have been fixed since 0.8.0.

The development on rootpy has been rather quiet lately for a variety of reasons. One being that there has been less contribution from the community in general. Another being that I've focussed more on root_numpy in the past several months. And finally, ROOT 6 has been a bit bumpy thus far and I'd rather let the dust settle there before making any lasting design choices in rootpy. It has been like a game of cat-and-mouse trying to maintain stability in ROOT 6.

My goal is to have rootpy declared "stable" with a pretty much frozen API by version 1.0.0. I'd also like to either get rid of or rewrite some of the more experimental corners of rootpy.
